engine/script/src/script_json.cpp

```diff
@@ -24,8 +24,11 @@
 
 extern "C"
 {
-#include <lua/lua.h>
-#include <lua/lauxlib.h>
+    #include <lua/lua.h>
+    #include <lua/lauxlib.h>
+
+    // Defined in luacjson/cjson.c
+    int lua_cjson_encode(lua_State *L);
```

Comment:
The entry point for the library required us to push both encode and decode functions to a table, so we've added an extra function in the cjson file that exposes the functionality we need instead.